sklearn/feature_extraction/text.py

```diff
@@ -269,21 +269,41 @@ def get_stop_words(self):
         return _check_stop_list(self.stop_words)
 
     def _check_stop_words_consistency(self, stop_words, preprocess, tokenize):
+        """Check if stop words are consistent
+
+        Returns
+        -------
+        is_consistent : True if stop words are consistent with the preprocessor
+                        and tokenizer, False if they are not, None if the check
+                        was previously performed, "error" if it could not be
+                        performed (e.g. because of the use of a custom
+                        preprocessor / tokenizer)
+        """
         # NB: stop_words is validated, unlike self.stop_words
         if id(self.stop_words) != getattr(self, '_stop_words_id', None):
```

Comment:
this is getting very nested. Perhaps reverse the condition here and return early instead

Reply:
Thanks for the review. Good point, reduced the nesting.